modules/administration-guide/partials/proc_migration_olm_stable_channel_to_stable_all_namespace.adoc

```diff
@@ -0,0 +1,271 @@
+
+[id="migration-olm-stable-channel-to-stable-all-namespace_{context}"]
+= Migration {prod} from "che-server" workspace engine to "dev-workspace" using OLM 
+
+For {prod} instances installed using Operator lifecycle manager(OLM) with "stable" channel
+there is opportunity to make migration with new channel "tech-preview-stable-all-namespaces".
+"stable" OLM channel provides {prod} with old "che-server" engine.
+This engine is not under active development any more.
+"tech-preview-stable-all-namespaces" channel provides {prod-short} with modern "dev-workspace" engine.
```

Comment:
I'm not sure if the official engine naming is `dev-workspace`. In any case we need a link to the new engine documentaiton